index.php

```diff
@@ -21,12 +16,11 @@
 
 $db = DB::getDB();
 $biscuits = BiscuitHydrator::getBiscuits($db);
-$biscuitDisplay = BiscuitViewHelper::displayAllBiscuits($biscuits, $dunkedFlunkedData['dunkedFlunked']);
-
+$biscuitDisplay = BiscuitViewHelper::displayAllBiscuits($biscuits, $dunkedFlunkedData);
 $mostDunked = BiscuitDataProcessor::mostDunked($biscuits);
 $mostFlunked = BiscuitDataProcessor::mostFlunked($biscuits);
-
 ?>
+
 <html lang="en-gb">
 <head>
     <title>Dunkin' Bisquids</title>
```

Comment:
can't add a comment further below: it is for the h6 heading : Error for h6 heading in WAVE. skipping headings. should be in order.